sandbox/RMSD_test_pentah5.ipynb

```diff
@@ -3,7 +3,11 @@
   {
    "cell_type": "code",
```

Comment:
We should probably remove this directory

Reply:
Agree. Will be removed in next pull request.